CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+Please open a new issue or new pull request for bugs, feedback, or new features you would like to see. If there is an issue you would like to work on, please leave a comment and we will be happy to assist. New contributions and contributors are very welcome!
+
+New to github or open source projects? If you are unsure about where to start or haven't used github before, please feel free to contact the package maintainers. Want more information about how to make a contribution? 
+
+Feedback and feature requests? Is there something missing you would like to see? Please open an issue or send an email to the maintainers. This package follows the Spacetelescope Code of Conduct and strives to provide a welcoming community to all of our users and contributors.
```

Comment:
Thanks, @sosey.  I would recommend linking "Code of Conduct" to `CODE_OF_CONDUCT.md` here.

Reply:
done.